### ISSUE TITLE
Einfache Heizungsregler f Haupt und Nebenraum beschaffen

### ISSUE THREAD

**Issue by Bfritz0815** [raumzeitlabor/rzl-tuwat#116]
Bevor wir komplexe steuerbare Dinge anschaffen, wären einfache zeitschaltbare temperatur-regelbare Thermostate sinnvoll. @felicitus baut gerade welche in ihrem neuen Büro ein. Da können wir also Erfahrungen dazu bekommen. Sie meinte, ein son Gerät kostet ca 10€. Bei unseren Heizungskosten läge da ne Anschaffung nahe.

**Comment by Bfritz0815** [comment]
@felicitus welche marke/modell ist denn bei dir verbaut?

**Comment by Bfritz0815** [comment]
also ne kurze recherche hat ergeben, dass Baumarkt 2 sinnvolle im Angebot hat:
1. https://www.bauhaus.info/thermostatventile/admiral-energiesparregler-comet-/p/20669366
2. https://www.bauhaus.info/thermostatventile/energiesparregler-comet-blue-/p/23557433

Der  zwote lässt sich per bluetooth steuern, kostet aber nen Tick mehr. (20€ statt 15€)

Wir bräuchten insgsamt 6 Regler, also:
Modell 1 =   90€
Modell 2 = 120€

Für schnell-mal-eben wäre also der erste am Besten. Meinungen?

Ich däte das dann mal demnächst aufs Plenum bringen (in ein paar Wochen, damit Zeit zum vorab-Diskutieren ist).

**Comment by s1lvester** [comment]
@abrock school of engeneering - aus zeitmangel eingestellt?

**Comment by abrock** [comment]
Steller und Kabel liegen zu allen Heizkörpern im Hauptraum und zu keinem Heizkörper im Nebenraum. Ich weiß gerade nicht ob die Platine den Anschluss von noch zwei Stellern hergibt, schlimmstenfalls könne man aber auch im Hauptraum 2x2 Steller koppeln.
Im Nebenraum gibt es auch keine Temperatursensoren, da müsste man also ein paar DS18S20 (WIMRE) verbauen.

Firmware und Schematics liegen hier:
https://github.com/raumzeitlabor/heizungssteuerung

Ich habe kurz in die main.c reingekuckt und keinen Code gefunden, der die Heizungssteller schalten würde, muss also jemand noch einbauen (halt auf seriell auf Kommandos warten und umsetzen).
Idealerweise würde die Steuerung in Abwesenheit von irgendwelchen externen Befehlen ein Minimalprogramm autark fahren, z.B. "Versuche >= 20°C an Sensoren XYZ zu erreichen", damit ohne Netzwerk der Raum nicht kalt wird.

Aus meiner Sicht ist das überschaubarer Aufwand, jedenfalls für den Hauptraum.

**Comment by uwap** [comment]
Mal eine ganz andere Frage: Macht es mehr Sinn als ein "Die Heizung ist komplett aus im Sommer und die Heizung heizt auf voller Stärke im Winter" Modus?

**Comment by Bfritz0815** [comment]
Ja, ich glaub, es macht Sinn: So schnell wie die Temperatur im Raum abfällt, macht es keinen Sinn, leere Räume fortlaufend zu beheizen. Wenn wir ne Gute Steuerung hätten, könnte die z.B. auch abschalten, sobald der Raum abgeschlossen wird etc.
So oft, wie ich als Erster reinkomme und die Heizung im Raum (und vor Allem im Nebenraum) bollert wie blöde, sollte sich das massiv lohnen. Ich versteh ja, dass Leute gerne den Raum vorgeheizt haben wollen, aber da reicht auch eine Stunde Vorheizen statt einem Tag. Allein daraus ergibt sich eine deutliche Ersparnis.

Worüber ich mir auch Gedanken mache ist, dass die Lösung, die ich vorschlage 120€ kostet, aber schnell aufzusetzen wäre. Das Projekt Heizungssteuerung eiert seit anno-dazumal ohne activity rum: https://github.com/raumzeitlabor/heizungssteuerung/graphs/contributors
Daher könnte man das auch so sehen: Wieviel cash werden wir unnötigerweise zum Schlot rausjagen, bevor das Heizungssteuerungs-Projekt flügge wird? Mehr oder weniger als 120€?

PS:
Was das Projekt betrifft: ich weiss zwar nicht, was sich wer wann für Gedanken zur Heizungssteuerung gemacht hat, aber nach dem was ich so mitgekriegt hatte, gings da weniger ums Steuern, sondern ums Messen. Kann aber sein, dass ich mich da irre.

**Comment by felicitus** [comment]
@Bfritz0815 Tipp: frag mal DaFo und AlexanderB direkt. Die beiden lesen vermutlich nicht aktiv hier mit und können dir wertvolle Infos über den Stand geben ^^

**Comment by abrock** [comment]
Eine kurze Geschichte der Heizungsregelung:

1. a) Kübler baut eine Heizung ein. Diese funktioniert furchtbar schlecht in den ersten ~2 Jahren.
1. b) Gleichzeitig bauen Dafo, Nessi und ich eine Platine mit uC der Temperatursensoren im ganzen Raum ausliest und im Prinzip auch den Status der Fenster lesen kann (hat irgendwelche Wackelkontakte, hat mich aber nicht weiter gestört).
Die Platine kann auch die Heizungssteller schalten, aber die sind noch nicht angeschlossen weil die erst angeschlossen werden dürfen wenn sie an der Heizung montiert sind.

Ich habe die nie angeschlossen weil ich es für Zeitverschwendung hielt eine nicht funktionierende Heizung irgendwie regeln zu wollen.

2. Robert stellt den Druck der Heizung so ein dass sie auch bei und funktioniert.

Das hatte zur Folge dass der Raum sinnvoll geheizt wurde außer Abends (Nachtabsenkung) um am WE (WE-Absenkung).
Ich kam dann mehrfach unter der Woche mittags / nachmittags in den Raum nur um festzustellen dass alle Heizungsregler auf 5 standen und der Raum brütend heiß war.
Ich habe mehrfach auf der ML erklärt was ein Heizungsregler ist und warum es völlig sinnlos ist die Regler auf 5 zu stellen, dass davon der Raum abends nicht wärmer wird sondern lediglich tagsüber wenn niemand da ist zur Sauna wird und sinnlos Heizöl verbraucht.
Ich habe nie eine Antwort bekommen, es kam bei den Verantwortlichen aber auch nicht an, denn geändert hat sich nichts.

Ich nehme an dass Leute, die sich so verhalten auch versuchen würden jedwede digitale Heizungsregelung irgendwie zu überbrücken so dass die Heizungen dauerhaft auf maximaler Leistung stehen und unter diesen Bedingungen sehe ich es als sinnlos irgendwelche Änderungen am aktuellen Zustand vorzunehmen, es ändert nichts am Verbrauchsprofil.

Wenn irgendjemand die von mir verbaute Hardware benutzen will um die Heizung zu regeln kann ich gerne Fragen dazu beantworten / einen WE-Tag im RZL verbringen um mitzuhelfen aber um es alleine zu machen steht es zu weit unten auf der Prioritätenliste.
